### PR TITLE
 [TR] translation of 10-3.md

### PR DESCRIPTION
**Pull request by EmreOzkose**

### CODE REVIEW

File: docs/tr/week10/10-1.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+date: 6 Apr 2020
+translator: Yunus Emre Özköse
+translation-date: 24 Jun 2020
+---
+
+<!--
+title: Self-Supervised Learning - Pretext Tasks
+-->
+
+=======
```
Comment (Atcold) [inline docs/tr/week10/10-1.md:17]: Remove this stuff, plz.

File: docs/tr/week10/10-1.md [hunk 2]
```diff
@@ -9,6 +23,7 @@ translator: EmreOzkose (Yunus Emre Özköse)
 translation-date: 24 Jun 2020
 ---
 
+>>>>>>> 61fbfd3e023e318eb24fad36eabfd705d9a4d598
```
Comment (Atcold) [inline docs/tr/week10/10-1.md:26]: Remove.

File: docs/tr/week10/10-3.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
+---
+lang-ref: ch.10-3
+lang: tr
+title: The Truck Backer-Upper
```
Comment (Atcold) [inline docs/tr/week10/10-3.md:4]: Translation needed.

File: docs/tr/week10/10-3.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+lecturer: Alfredo Canziani
+authors: Muyang Jin, Jianzhi Li, Jing Qian, Zeming Lin
+date: 7 Apr 2020
+translator: Yunus Emre Özköse
+translation-date: 19 July 2020
```
Comment (Atcold) [inline docs/tr/week10/10-3.md:9]:
```suggestion
translation-date: 19 Jul 2020
```

File: docs/tr/week10/10.md [hunk 5]
```diff
@@ -1,10 +1,21 @@
...
+<<<<<<< HEAD
+title: 10. Hafta
+translator: Yunus Emre Özköse
+translation-date: 24 Jun 2020
+---
+
+<!--
+title: Week10
+-->
+=======
```
Comment (Atcold) [inline docs/tr/week10/10.md:12]: Remove.

File: docs/tr/week10/10.md [hunk 6]
```diff
@@ -1,10 +1,21 @@
...
 title: Week 10
 translator: EmreOzkose (Yunus Emre Özköse)
 translation-date: 24 Jun 2020
 ---
 
+>>>>>>> 61fbfd3e023e318eb24fad36eabfd705d9a4d598
```
Comment (Atcold) [inline docs/tr/week10/10.md:18]: Remove.

**Comment by EmreOzkose** [comment]
I fixed them and pushed changes.

**Comment by Atcold** [comment]
What happened here? 😮

**Pull request closed without merging**